apps/server/src/modules/oauth-provider/controller/oauth-provider.controller.ts

```diff
@@ -1,9 +1,11 @@
 import { Configuration } from '@hpi-schul-cloud/commons/lib';
 import { Body, Controller, Delete, Get, NotImplementedException, Param, Patch, Post, Put, Query } from '@nestjs/common';
-import { Authenticate } from '@src/modules/authentication/decorator/auth.decorator';
+import { Authenticate, CurrentUser } from '@src/modules/authentication/decorator/auth.decorator';
 import { OauthProviderLogoutFlowUc } from '@src/modules/oauth-provider/uc/oauth-provider.logout-flow.uc';
 import { OauthProviderResponseMapper } from '@src/modules/oauth-provider/mapper/oauth-provider-response.mapper';
 import { RedirectResponse } from '@shared/infra/oauth-provider/dto';
+import { OauthClient } from '@shared/infra/oauth-provider/dto/index';
```

Comment:
/index doesnt need to be specified, this is the same as the line above

Reply:
Done